sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/AppendBlobRequestConditions.java

```diff
@@ -82,6 +82,18 @@ public AppendBlobRequestConditions setLeaseId(String leaseId) {
         return this;
     }
 
+    /**
+     * Optionally applies the SQL statement to the tags of the blob.
+     *
+     * @param ifTags The SQL statement that apply to the tags of the blob.
+     * @return The updated BlobRequestConditions object.
+     */
+    @Override
+    public AppendBlobRequestConditions setIfTags(String ifTags) {
```

Comment:
Maybe we should call the parameter `ifTagsMatch` or something? `ifTags` reads to me like it's just checking if there are tags present

Reply:
I like that idea

Reply:
There's a lot of over conditions that can be checked related to tags.  In .NET, we went with [TagConditions](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/storage/Azure.Storage.Blobs/src/Models/BlobLeaseRequestConditions.cs#L14)

Reply:
Changed to tagConditions